tests/handformer.test.cpp

```diff
@@ -4,15 +4,17 @@
 #include "handformer.h"
 #include "piecetype.h"
 
-using namespace Mahjong;
+using Mahjong::Piece;
+using Mahjong::HandFromNotation;
+using Mahjong::IsValidNotation;
+using Mahjong::HandToNotation;
```

Comment:
This is fine but fyi you can also do
```
using Mahjong::Piece, Mahjong::HandFromNotation, ...;
```
I think this is cleaner but i don't mind the current form.

Reply:
For module usages >= 4 I believe the separate lines are cleaner, akin to javascript importing.